htdocs/config2/locale.inc.php

```diff
@@ -39,32 +39,6 @@
 $opt['template']['locales']['NL']['flag'] = 'images/flag/NL.png';
 $opt['template']['locales']['NL']['name'] = 'Nederlands';
 
-/*
-    Disabling unused locales speeds up template precompilation (clear-webcache.php).
-
-    $opt['template']['locales']['SV']['show'] = true; // sv_SE
-    $opt['template']['locales']['SV']['flag'] = 'images/flag/SE.png';
-    $opt['template']['locales']['SV']['name'] = 'Svenska';
-    $opt['template']['locales']['NO']['show'] = true; // no_NO
-    $opt['template']['locales']['NO']['flag'] = 'images/flag/NO.png';
-    $opt['template']['locales']['NO']['name'] = 'Norsk';
-    $opt['template']['locales']['PL']['show'] = true;
-    $opt['template']['locales']['PL']['flag'] = 'images/flag/PL.png';
-    $opt['template']['locales']['PL']['name'] = 'Polski';
-    $opt['template']['locales']['RU']['show'] = true;
-    $opt['template']['locales']['RU']['flag'] = 'images/flag/RU.png';
-    $opt['template']['locales']['RU']['name'] = 'Русский';
-    $opt['template']['locales']['DA']['show'] = true; // da_DK
-    $opt['template']['locales']['DA']['flag'] = 'images/flags/dk.png';
-    $opt['template']['locales']['DA']['name'] = 'Danske';
-    $opt['template']['locales']['PT']['show'] = true; // pt_PT
-    $opt['template']['locales']['PT']['flag'] = 'images/flags/pt.png';
-    $opt['template']['locales']['PT']['name'] = 'Portuguesa';
-    $opt['template']['locales']['JA']['show'] = true; // ja_JP
-    $opt['template']['locales']['JA']['flag'] = 'images/flag/JP.png';
-    $opt['template']['locales']['JA']['name'] = '日本語';
-*/
-
```

Comment:
Please let's keep the out-commented code in locale.inc.php. It's not obsolete but regularly used when working on translations, either by enabling/disabling it here or copying it to the local settings.inc.php.